js/src/components/google-mc-account-card/connected-google-mc-account-card.js

```diff
@@ -85,6 +97,44 @@ const ConnectedGoogleMCAccountCard = ( {
 		removeNotice( notice.id );
 	};
 
+	const disableNotifications = async () => {
+		const { notice } = await createNotice(
+			'info',
+			__(
+				'Disabling the new Product Sync feature, please wait…',
+				'google-listings-and-ads'
+			)
+		);
+
+		try {
+			await fetchDisableNotifications();
+			invalidateResolution( 'getGoogleMCAccount', [] );
+		} catch ( error ) {
+			createNotice(
+				'error',
+				__(
+					'Unable to disable new product sync. Please try again later.',
+					'google-listings-and-ads'
+				)
+			);
+		}
+
+		removeNotice( notice.id );
+	};
+
+	// Show the button if the status is "approved" and the Notification Service is not hidden.
+	const showDisconnectNotificationsButton =
+		! hideNotificationService &&
+		googleMCAccount.wpcom_rest_api_status === 'approved';
+
+	// Show the error if the status is set but is not "approved" and the Notification Service is not hidden.
+	const showErrorNotificationsNotice =
+		! hideNotificationService &&
+		googleMCAccount.wpcom_rest_api_status &&
+		googleMCAccount.wpcom_rest_api_status !== 'approved';
```

Comment:
Same as https://github.com/woocommerce/google-listings-and-ads/pull/2311/files#r1539571082